docker-compose.yml

```diff
@@ -3,10 +3,12 @@ version: '3.7'
 services:
     dbt_image:
       image: fishtownanalytics/dbt:1.0.0
+      env_file:
+        - .dbt.env
       environment:
         DBT_PROFILES_DIR: /usr/local/dbt_profiles/
         SNOWFLAKE_LOAD_DATABASE: "RAW"
-        SNOWFLAKE_TRANSFORM_WAREHOUSE: "TRANSFORM_XS"
+        SNOWFLAKE_TRANSFORM_WAREHOUSE: "TRANSFORM_L"
```

Comment:
is it to reduce the build time of dev models? for building just one or two models its a lot of compute.

Reply:
I checked available warehouses and it was the only one available with prefix `TRANSFORM`. I can try one of the others.